### PR TITLE
chore: add .markdownlint.json from mdn/content

### PR DESCRIPTION
**Pull request by nschonni**
I'm submitting some cleanups separately to align the content, but the CI job doesn't currently run across the repo right now

### CODE REVIEW

**Comment by yin1999** [comment]
should we only let the workflow run on changed markdown files: https://github.com/mdn/translated-content/blob/82937239e0b147b6206bac8a7741f549635be61b/.github/workflows/markdown-lint.yml#L24-L26

File: .github/workflows/markdown-lint.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
     paths:
       - '*.md'
```
Comment (nschonni) [inline .github/workflows/markdown-lint.yml:8]: @yin1999 right now this action only runs when a markdown file in the root is touched, not for any of the actual translated files

**Comment by cw118** [comment]
Poking @mdn/core-yari-dev for a review :)

**Comment by schalkneethling** [comment]
Hey @nschonni, thank you for opening this. Could we perhaps use the one here? https://github.com/mdn/workflows/blob/main/.github/workflows/markdown-lint.yml (https://github.com/mdn/workflows#markdown-lint)

If we need to make some changes to make it more customisable so it can be used here, that would be excellent.

**Comment by nschonni** [comment]
@schalkneethling you could do the `uses:` thing like you have for some of the other actions, but that is separate from this change. Either way the current globs are incorrect and only touching the README file and none of the actual content files. That is the same issue being tracked in the discussion.
This only adds the same config file as used in the content repo, so the same rules are picked up for the current single file run, plus flags the issues for contributors that have a Markdownlint plugin in their IDE. This config file can't live in the shared repo for that reason.

**Comment by schalkneethling** [comment]
> @schalkneethling you could do the `uses:` thing like you have for some of the other actions, but that is separate from this change. Either way the current globs are incorrect and only touching the README file and none of the actual content files. That is the same issue being tracked in the discussion.
> This only adds the same config file as used in the content repo, so the same rules are picked up for the current single file run, plus flags the issues for contributors that have a Markdownlint plugin in their IDE. This config file can't live in the shared repo for that reason.

💯 Thank you for the feedback.

**Pull request merged**